scripts-local/init_stride.sh

```diff
@@ -42,6 +42,7 @@ for NODE_NAME in stride stride2 stride3 stride4 stride5; do
     # NOTE: If you add new epochs, these indexes will need to be updated
     jq '.app_state.epochs.epochs[$epochIndex].duration = $epochLen' --arg epochLen $DAY_EPOCH_LEN --argjson epochIndex $DAY_EPOCH_INDEX  $main_config > json.tmp && mv json.tmp $main_config
     jq '.app_state.epochs.epochs[$epochIndex].duration = $epochLen' --arg epochLen $STRIDE_EPOCH_LEN --argjson epochIndex $STRIDE_EPOCH_INDEX $main_config > json.tmp && mv json.tmp $main_config
+    jq '.app_state.epochs.epochs[$epochIndex].duration = $epochLen' --arg epochLen $MINT_EPOCH_LEN --argjson epochIndex $MINT_EPOCH_INDEX $main_config > json.tmp && mv json.tmp $main_config
```

Comment:
rather than use an index for these, you can do:
```
jq '(.app_state.epochs.epochs[] | select(.identifier=="day") ).duration = $epochLen' --arg epochLen $DAY_EPOCH_LEN $main_config > json.tmp && mv json.tmp $main_config
jq '(.app_state.epochs.epochs[] | select(.identifier=="stride_epoch") ).duration = $epochLen' --arg epochLen $STRIDE_EPOCH_LEN $main_config > json.tmp && mv json.tmp $main_config
jq '(.app_state.epochs.epochs[] | select(.identifier=="mint") ).duration = $epochLen' --arg epochLen $MINT_EPOCH_LEN $main_config > json.tmp && mv json.tmp $main_config
```
then you can remove DAY_EPOCH_INDEX, STRIDE_EPOCH_INDEX, MINT_EPOCH_INDEX from vars